packages/cli-repl/test/e2e-fle.spec.ts

```diff
@@ -762,7 +762,7 @@ describe('FLE tests', () => {
       await shell.executeLine(`db = db.getSiblingDB('${dbname}')`);
       await shell.executeLine('db.version = () => \'6.0.0\'');
       const result = await shell.executeLine('db.createCollection(\'test\', { encryptedFields: { fields: [] } });');
-      expect(result).to.match(/Your featureCompatibilityVersion is .+, which does not support Queryable Encryption/);
+      expect(result).to.match(/Upgrade server to use Queryable Encryption./);
```

Comment:
I think we can remove the entire `_improveErrorMessageForLowServerVersionForQE()` logic then?